setup.cfg

```diff
@@ -30,22 +36,45 @@ universal=1
 zip_safe = false
 py_modules = multicast
 test_suite = tests
-python_requires = >=3.7, !=3.11.*
+python_requires = >=3.7, !=3.13.*
 setup_requires =
-    setuptools>=45.0.0
-    wheel>=0.37.0
+  setuptools>=45.0.0
+  wheel>=0.37.0
+  build>=1.2.1
 
 scripts =
   multicast/__main__.py
+  setup.py
```

Comment:
:thinking: is this documented in a PEP (and ok with PEP-517) and is this correct?
- [ ] open ticket

Reply:
Opened #58 🏷️ for this